psi4/driver/procrouting/sapt/sapt_proc.py

```diff
@@ -432,7 +460,23 @@ def sapt_dft(dimer_wfn, wfn_A, wfn_B, sapt_jk=None, sapt_jk_B=None, data=None, p
                                                          cache,
                                                          do_print=True)
     data.update(mp2_disp)
-    core.timer_off("SAPT(DFT):SAPT(DFT):disp")
+
+    # Exchange-dispersion scaling
+    exch_disp_scheme = core.get_option("SAPT", "SAPT_DFT_EXCH_DISP_SCALE_SCHEME")
+    core.print_out("    %-33s % s\n" % ("Scaling Scheme", exch_disp_scheme))
+    if exch_disp_scheme == "NONE":
+        data["Exch-Disp20,r"] = data["Exch-Disp20,u"]
+    elif exch_disp_scheme == "FIXED":
+        exch_disp_scale = core.get_option("SAPT", "SAPT_DFT_EXCH_DISP_FIXED_SCALE")
+        core.print_out("    %-28s % 10.3f\n" % ("Scaling Factor", exch_disp_scale))
+        data["Exch-Disp20,r"] = exch_disp_scale * data["Exch-Disp20,u"]
+    elif exch_disp_scheme == "DISP":
+        exch_disp_scale = data["Disp20"] / data["Disp20,u"]
+        data["Exch-Disp20,r"] = exch_disp_scale * data["Exch-Disp20,u"]
+    core.print_out(print_sapt_var("Exch-Disp20,r", data["Exch-Disp20,r"], short=True) + "\n")
```

Comment:
> This will match the printout in the sapt dft summary below, right? The "Est." should help with matching quantities and emphasize that it's not explicitly computed. Thanks for adding the options status.

Yes. I'll manually modify this since I'd like the program not to print out this line if exch_disp_scheme is NONE.